nm/src/nm.c

```diff
@@ -426,7 +426,7 @@ _j4status_nm_device_update(J4statusPluginContext *context, J4statusNmSection *se
             if ( context->formats.up_eth_tokens & TOKEN_FLAG_UP_ETH_ADDRESSES )
                 data.addresses = _j4status_nm_device_get_addresses(context, device);
 
-            data.speed = nm_device_ethernet_get_speed(NM_DEVICE_ETHERNET(device)) * 1000;
+            data.speed = nm_device_ethernet_get_speed(NM_DEVICE_ETHERNET(device)) * 1000000;
```

Comment:
Good catch, did it change with libnm or was it broken before already?

Reply:
I think it came with libnm. Previously, it worked.

Reply:
Then it should just be in the same commit, no need to split here.